docs/source/api/python/mantid/api/SpectrumInfo.rst

```diff
@@ -4,6 +4,165 @@
 
 This a python binding to the C++ class Mantid::API::SpectrumInfo.
 
+--------  
+Purpose 
+-------- 
+The purpose of the ``SpectrumInfo`` object is to allow the user to access information about the spectra being used in an experiment. The ``SpectrumInfo`` object can be used to access information such as the number of spectra, the absolute position of a spectrum as well as the distance from the sample to the source. There are many other methods available as well. 
+
+A Spectrum corresponds to (a group of) one or more detectors. However if no instrument/ beamline has been set then it could also technically be zero or more detectors. An example test case details this below.  
```

Comment:
If there is no instrument, it is always `zero`, not `technically zero or more`? Even if there is an instrument, a spectrum can have zero detectors.

Reply:
Indeed. This has been discussed, and Bhuvan will be correcting it.